Framework/Core/src/DataOutputDirector.cxx

```diff
@@ -351,8 +350,21 @@ std::tuple<std::string, std::string, std::string, float, int> DataOutputDirector
     }
   }
 
+  itemName = "offset";
+  if (dodirItem.HasMember(itemName)) {
+    if (dodirItem[itemName].IsNumber()) {
+      offset = dodirItem[itemName].GetUint64();
+      setDFOffset(offset);
+    } else {
+      LOGP(error, "Check the JSON document! Item \"{}\" must be a number!", itemName);
+      return memptyanswer;
+    }
+  }
+
   itemName = "OutputDescriptors";
   if (dodirItem.HasMember(itemName)) {
+    std::string slh("/");
```

Comment:
static / const, no?

Reply:
The whole thing needs a refactoring pass, but it is not critical.